From bed02277139fc2d53f3a1bcd5918a1dc7bd544e6 Mon Sep 17 00:00:00 2001
From: Daniel Hsu <Daniel-Hsu@quantatw.com>
Date: Thu, 2 Jul 2026 19:52:18 +0800
Subject: [PATCH] configurations: ocp: Remove low temperature threshold for CX7
 NIC sensor

Remove the lower critical threshold from the CX7 NIC temperature
sensor configuration to avoid false low temperature events caused by
transient or invalid sensor readings.
---
 configurations/ocp/cx7_ocp.json | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/configurations/ocp/cx7_ocp.json b/configurations/ocp/cx7_ocp.json
index 480d37c..b4296c9 100644
--- a/configurations/ocp/cx7_ocp.json
+++ b/configurations/ocp/cx7_ocp.json
@@ -21,14 +21,6 @@
             "Name": "NIC$bus % 2 TEMP_C",
             "Name1": "NIC$bus % 2 TEMP_C",
             "Thresholds": [
-                {
-                    "Direction": "less than",
-                    "Hysteresis": 0.1,
-                    "Index": 2,
-                    "Name": "lower critical",
-                    "Severity": 1,
-                    "Value": 5
-                },
                 {
                     "Direction": "greater than",
                     "Hysteresis": 2.1,
--
2.53.0

